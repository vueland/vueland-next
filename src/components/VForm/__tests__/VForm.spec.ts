import { mount, VueWrapper } from '@vue/test-utils'
import { VForm } from '../index'

describe('VForm', () => {
  let mountFunction: (options?: any) => VueWrapper<any>

  beforeEach(() => {
    mountFunction = (options = {}) => mount(VForm, { ...options })
  })

  it('should mount component and match snapshot', () => {
    const cmp = mountFunction()

    expect(cmp.html()).toMatchSnapshot()
  })
})
