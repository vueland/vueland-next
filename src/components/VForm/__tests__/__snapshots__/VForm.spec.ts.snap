// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VForm should mount component and match snapshot 1`] = `
<span class="v-form">
</span>
`;
